packages/main/src/components/Text/index.tsx

```diff
@@ -22,21 +23,11 @@ export interface TextProps extends CommonProps {
   width?: CSSProperties['width'];
 }
 
-interface TextInternalProps extends TextProps, ClassProps {}
+const useStyles = createUseStyles<JSSTheme, string>(TextStyles, { name: 'Text ' });
```

Comment:
Did you add the space after Text intentionally?

Reply:
No 🙈